forge-gui/res/cardsfolder/upcoming/ral_and_the_implicit_maze.txt

```diff
@@ -0,0 +1,13 @@
+Name:Ral and the Implicit Maze
+ManaCost:3 R R
+Types:Enchantment Saga
+K:Chapter:3:DBDamageAll,DBImpulseDraw,DBToken
+SVar:DBDamageAll:DB$ DamageAll | ValidCards$ Creature.OppCtrl,Planeswalker.OppCtrl | NumDmg$ 2 | SpellDescription$ CARDNAME deals 2 damage to each creature and planeswalker your opponents control.
+SVar:DBImpulseDraw:DB$ Dig | Cost$ Discard<1/Card> | Defined$ You | DigNum$ 2 | ChangeNum$ All | DestinationZone$ Exile | SubAbility$ DBEffect | RememberChanged$ True | SubAbility$ DBEffect | SpellDescription$ You may discard a card. If you do, exile the top two cards of your library. You may play them until the end of your next turn.
+SVar:DBEffect:DB$ Effect | RememberObjects$ RememberedCard | StaticAbilities$ STPlay | SubAbility$ DBCleanup | ForgetOnMoved$ Exile | Duration$ UntilTheEndOfYourNextTurn
+SVar:STPlay:Mode$ Continuous | MayPlay$ True | EffectZone$ Command | Affected$ Card.IsRemembered | AffectedZone$ Exile | Description$ You may play them until the end of your next turn.
+SVar:DBCleanup:DB$ Cleanup | ClearRemembered$ True
+SVar:DBToken:DB$ Token | TokenAmount$ 1 | TokenScript$ spellgorger_weird | TokenOwner$ You | SpellDescription$ Create a Spellgorger Weird token. (It's a {2}{R} 2/2 Weird creature with "Whenever you cast a noncreature spell, put a +1/+1 counter on Spellgorger Weird.")

```

Comment:
This one needs the Token-Script from the other MR

Reply:
indeed, I think we can just keep this open until then
I've messed with enough commits for today :)